src/main/scala/jio/package.scala

```diff
@@ -29,19 +27,45 @@ package object jio extends JioFiles {
   }
 
   implicit class PathOps(val p: Path) extends AnyVal {
+
     def /(name: String): Path = p resolve name
 
     def mkdir(attrs: AnyFileAttr*): Path = createDirectory(p, attrs: _*)
     def exists                           = p.toFile.exists
-    def isSymbolicLink()                 = Files isSymbolicLink p
-    def readSymbolicLink()               = Files readSymbolicLink p
-    def readAllBytes()                   = Files readAllBytes p
-    def list(): Vector[Path]             = Files list p toVector
+    def isFile                           = Files.isRegularFile(p, NOFOLLOW_LINKS)
+    def isDirectory                      = Files.isDirectory(p, NOFOLLOW_LINKS)
+    def isSymbolicLink                   = Files isSymbolicLink p
+    def readSymbolicLink                 = Files readSymbolicLink p
+    def allBytes                         = Files readAllBytes p
+    def list: Vector[Path]               = (Files list p).toVector
     def filename: String                 = p.getFileName.toString
+    def blockSize: Long                  = 512 // FIXME
+    def blockCount: Long                 = (attributes.size + blockSize - 1) / blockSize
+    def size: Long                       = attributes.size
+    def atime: Long                      = attributes.lastAccessTime.toMillis
+    def mtime: Long                      = attributes.lastModifiedTime.toMillis
```

Comment:
This might be inefficient as we call the attributes method for a few of those properties. For now I however think performance should not be a concern.